src/main/cms/cms.c

```diff
@@ -901,16 +901,16 @@ static void cmsUpdate(uint32_t currentTimeUs)
     lastCalledMs = currentTimeMs;
 }
 
-void cmsHandler(uint32_t currentTime)
+void cmsHandler(timeUs_t currentTimeUs)
 {
     if (cmsDeviceCount < 0)
         return;
 
-    static uint32_t lastCalled = 0;
+    static timeUs_t lastCalled = 0;
 
-    if (currentTime >= lastCalled + CMS_UPDATE_INTERVAL_US) {
-        lastCalled = currentTime;
-        cmsUpdate(currentTime);
+    if (currentTimeUs >= lastCalled + CMS_UPDATE_INTERVAL_US) {
```

Comment:
It's safer to have `currentTimeUs - lastCalled >= CMS_UPDATE_INTERVAL_US` - imune to overflow. However we are now unlikely to have overflows :smile: